airbyte-integrations/connectors/destination-postgres/metadata.yaml

```diff
@@ -5,7 +5,7 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 25c5221d-dce2-4163-ade9-739ef790f503
-  dockerImageTag: 2.3.0
+  dockerImageTag: 2.3.1
```

Comment:
you need to rebump this, 2.3.1 got published earlier today 😅 https://github.com/airbytehq/airbyte/pull/43363